.github/workflows/auto-publish.yml

```diff
@@ -7,9 +7,13 @@ jobs:
     if: github.repository_owner == 'Fabulously-Optimized'
     runs-on: ubuntu-24.04
     environment: github-actions
+    permissions:
+      id-token: write
+      attestations: write
+      contents: write
```

Comment:
They probably don't need write, but I don't know if the categories of permissions are correct

Reply:
were these categories found/generated by that same library?

Reply:
Looked into it and turns out both git-repo-sync and CF/Modrinth needs write access to properly function, will update this.